src/sentry/static/sentry/app/views/settings/account/accountDetails.jsx

```diff
@@ -13,22 +16,12 @@ class AccountDetails extends AsyncView {
     return [['user', ENDPOINT]];
   }
 
-  handleSubmitSuccess = (change, model, id) => {
-    // special logic for linked password fields
-    if (id === 'passwordVerify' || id === 'password') {
-      model.setValue('password', '');
-      model.setValue('passwordVerify', '');
-    } else {
-      model.setValue(id, '');
-    }
+  handleSubmitSuccess = user => {
+    updateUser(user);
```

Comment:
This won't update avatar on sidebar when you exit settings because that component is not connected to `ConfigStore`

Reply:
would a reasonable followup be fixing that?

Reply:
or is it hard?

Reply:
ya there'll be a follow up, didn't want to expand this PR even more